NEWS.rst

```diff
@@ -5,6 +5,119 @@ User-Visible Changes in Tahoe-LAFS
+- The [storage] configuration section now accepts a boolean *anonymous* item to enable or disable anonymous storage access.  The default behavior remains unchanged. (`#3184 <https://github.com/tahoe-lafs/tahoe-lafs/issues/3184>`_)
+- Enable the helper when creating a node with `tahoe create-node --helper` (`#3235 <https://github.com/tahoe-lafs/tahoe-lafs/issues/3235>`_)
+
+
+Bug Fixes
+---------
+
+- refactor initialization code to be more async-friendly (`#2870 <https://github.com/tahoe-lafs/tahoe-lafs/issues/2870>`_)
+- Configuration-checking code wasn't being called due to indenting (`#2935 <https://github.com/tahoe-lafs/tahoe-lafs/issues/2935>`_)
+- refactor configuration handling out of Node into _Config (`#2936 <https://github.com/tahoe-lafs/tahoe-lafs/issues/2936>`_)
+- "tox -e codechecks" no longer dirties the working tree. (`#2941 <https://github.com/tahoe-lafs/tahoe-lafs/issues/2941>`_)
+- Updated the Tor release key, used by the integration tests. (`#2944 <https://github.com/tahoe-lafs/tahoe-lafs/issues/2944>`_)
+- `tahoe backup` no longer fails with an unhandled exception when it encounters a special file (device, fifo) in the backup source. (`#2950 <https://github.com/tahoe-lafs/tahoe-lafs/issues/2950>`_)
+- Magic-Folders now creates spurious conflict files in fewer cases.  In particular, if files are added to the folder while a client is offline, that client will not create conflict files for all those new files when it starts up. (`#2965 <https://github.com/tahoe-lafs/tahoe-lafs/issues/2965>`_)
+- The confusing and misplaced sub-command group headings in `tahoe --help` output have been removed. (`#2976 <https://github.com/tahoe-lafs/tahoe-lafs/issues/2976>`_)
+- The Magic-Folder frontend is now more responsive to subtree changes on Windows. (`#2997 <https://github.com/tahoe-lafs/tahoe-lafs/issues/2997>`_)
+- remove ancient bundled jquery and d3, and the "dowload timeline" feature they support (`#3228 <https://github.com/tahoe-lafs/tahoe-lafs/issues/3228>`_)
+
+
+Dependency/Installation Changes
+-------------------------------
+
+- Tahoe-LAFS no longer makes start-up time assertions about the versions of its dependencies.  It is the responsibility of the administrator of the installation to ensure the correct version of dependencies are supplied. (`#2749 <https://github.com/tahoe-lafs/tahoe-lafs/issues/2749>`_)
+- Tahoe-LAFS now depends on Twisted 16.6 or newer. (`#2957 <https://github.com/tahoe-lafs/tahoe-lafs/issues/2957>`_)
+
+
+Removed Features
+----------------
+
+- Fedora 27 is no longer tested as part of the project's continuous integration system. (`#2955 <https://github.com/tahoe-lafs/tahoe-lafs/issues/2955>`_)
+
+
+Other Changes
+-------------
+
+- Tahoe-LAFS now requires Twisted 18.4.0 or newer. (`#2771 <https://github.com/tahoe-lafs/tahoe-lafs/issues/2771>`_)
```

Comment:
Oops.  I guess this one belongs in the "Dependency/Installation Changes" section ... which also has an item saying Twitsed 16.6 is now required. :)